### PR TITLE
docs: Update docs for UAE VAT 201 Report

### PR DESCRIPTION
**Pull request by hasnain2808**
Docs For https://github.com/frappe/erpnext/pull/23447

**Report docs**
uae-vat-201-report

**Regional Fields**
image

**UAE Index**
image

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# UAE VAT 21 Report in ERPNext
+
+To generate GSTR3B Report in ERPNext navigate to <br>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md:3]: do you mean UAE VAT 21 report?

File: erpnext_documentation/www/docs/user/manual/en/regional/france/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!-- add-breadcrumbs -->
+# Requirements for France
+
+ERPNext aims to support local regulation for different regions in the world. In most cases ERPNext is very flexible so you can easily add Custom Fields and make Custom Reports to support the regulation of your region.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/france/index.md:4]: In most cases, ERPNext is very flexible, so you can easily add Custom Fields and make Custom Reports support the regulation of your region.

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<!-- add-breadcrumbs -->
+# Requirements for United Arab Emirates
+
+ERPNext aims to support local regulation for different regions in the world. In most cases ERPNext is very flexible so you can easily add Custom Fields and make Custom Reports to support the regulation of your region.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/index.md:4]: In most cases, ERPNext is very flexible, so you can easily add Custom Fields and make Custom Reports support the regulation of your region.

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+This section contains region specific features for United Arab Emirates.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/index.md:6]: region-specific

File: erpnext_documentation/www/docs/user/manual/en/regional/france/index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+This section contains region specific features for France.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/france/index.md:6]: region-specific

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md (hunk 1 above)
Comment (michellealva) [reply erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md:3]: To generate UAE VAT 21 report in ERPNext, go to <br>

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+**Accounting > Value-Added Tax (VAT UAE) > UAE VAT 21** <br>
+or simply search for UAE VAT 21 in awesomebar.
+
+<img class="screenshot" alt="UAE VAT 21 Report" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-21-report.png">
+
+If you want to print the report it can also be printed and viewed in by clicking on Print in Menu as shown below.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md:9]: To print the report, go to Menu > Print as shown in the screenshot below.

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+<img class="screenshot" alt="UAE VAT 21 Report Download" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-21-download.png">
+
+Note: To make sure the report is calculated accurately and correctly please make sure of the following things.
+
+1. Correct Emirate is selected for each Address.
+
+2. Nil rated or exempted Items have Is nil rated or Is Exempt checkbox checked in the item master.
+
+3. ALL VAT Accounts are selected in UAE VAT Settings.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md:19]: All VAT accounts...

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+<img class="screenshot" alt="UAE VAT Account Settings" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-account-settings.png">
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md:21]: crop the comment part (below part of the image). Take the screenshot only till add row button

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-setup.md [hunk 9]
```diff
@@ -44,3 +44,8 @@ ERPNext provides 2 default print foramt
 
 2. Detailed Tax Invoice
 <img class="screenshot" alt="Detailed Tax Invoice" src="{{docs_base_url}}/assets/img/regional/uae/detailed-invoice.png">
+
+### 5. Set-up VAT Accounts
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-setup.md:48]: Set up VAT Accounts

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-setup.md [hunk 10]
```diff
@@ -44,3 +44,8 @@ ERPNext provides 2 default print foramt
...
+Select the accounts that will be used for creating VAT invoices here.
+
+<img class="screenshot" alt="UAE VAT Account Settings" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-account-settings.png">
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-setup.md:51]: change the screenshot as advised above

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/regional/france/index.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+<!-- add-breadcrumbs -->
+# Requirements for France
+
+ERPNext aims to support local regulation for different regions in the world. In most cases, ERPNext is very flexible, so you can easily add Custom Fields and make Custom Reports support the regulation of your region.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/france/index.md:4]:
```suggestion
ERPNext aims to support local regulation for different regions in the world. In most cases, ERPNext is very flexible, so you can easily add custom fields and make custom reports to support the regulation of your region.
```

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/index.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+<!-- add-breadcrumbs -->
+# Requirements for United Arab Emirates
+
+ERPNext aims to support local regulation for different regions in the world. In most cases, ERPNext is very flexible, so you can easily add Custom Fields and make Custom Reports support the regulation of your region.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/index.md:4]:
```suggestion
ERPNext aims to support local regulation for different regions in the world. In most cases, ERPNext is very flexible, so you can easily add custom fields and make custom reports to support the regulation of your region.
```

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+# UAE VAT 201 Report in ERPNext
+
+To generate UAE VAT 201 Report in ERPNext go to <br>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md:3]:
```suggestion
To generate UAE VAT 201 Report in ERPNext go to:
```

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+**Accounting > Value-Added Tax (VAT UAE) > UAE VAT 201** <br>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md:4]:
```suggestion
> Accounting > Value-Added Tax (VAT UAE) > UAE VAT 201
```

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+or simply search for UAE VAT 201 in awesomebar.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md:5]:
```suggestion
or simply search for UAE VAT 201 in the awesomebar.
```

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+
+Note: To make sure the calculations in the report are correct, check the following:
+
+1. Correct Emirate is selected for each Address.
+
+2. Zero Rated or Exempted Items have "Is Zero Rated" or "Is Exempt" checkbox checked in the item master.
+
+3. ALL VAT accounts are selected in UAE VAT Settings.
+
+<img class="screenshot" alt="UAE VAT Account Settings" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-account-settings.png">
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md:20]: change this screenshot... crop it from bottom

Comment (hasnain2808) [reply erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md:20]: done

**Review by michellealva — APPROVED** [review]

File: erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+
+<img class="screenshot" alt="UAE VAT 201 Report Download" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-201-download.png">
+
+Note: To make sure the calculations in the report are correct, check the following:
+
+1. Correct Emirate is selected in Company Address and Sales Invoice. The Sales Invoice Emirate field is auto filled using the selected Company Address but is editable.
+
+2. Zero Rated or Exempted Items have "Is Zero Rated" or "Is Exempt" checkbox checked in the item master.
+
+3. ALL VAT accounts are selected in UAE VAT Settings.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md:19]:
```suggestion
3. All VAT accounts are selected in UAE VAT Settings.
```

**Review by michellealva — CHANGES_REQUESTED** [review]

**Comment by mohsinalimat** [comment]
UAE VAT issue.

https://github.com/frappe/erpnext/issues/24475

**Pull request merged**